resources/cluster-essentials/files/functions.serverless.crd.yaml

```diff
@@ -254,7 +254,7 @@ spec:
             runtime:
               enum:
               - nodejs12
-              - nodejs10
```

Comment:
Nodejs10 has to stay as allowed value in the status object. Otherwise editing existing function's runtime will fail.